Added etcd username and userpassword from config (#4205)
cortexproject/cortex: CHANGELOG.md

## master / unreleased

> * [ENHANCEMENT] Added etcd username and userpassword from config #4205

Reviewer: Order of entries in the changelog should be: `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`.

```suggestion
* [ENHANCEMENT] Etcd: Added username and password to etcd config. #4205
```

Reviewer 2: @ymhhh Could you fix this, please? We would be very happy to merge once done! Thanks!
